.yarn/versions/fa277a70.yml

```diff
@@ -0,0 +1,2 @@
+releases:
```

Comment:
This file is used by yarn to mark version upgrades. When you create a PR, you just use
```
yarn version major --deferred
```
to indicate this will be a major update (as I did for this PR). This info will be stored by yarn in this file.
When you eventually release a new version, you will run
```
yarn version apply
```
and it will combine all the version updates that are marked in this file.
This is a way to circumvent the need to check commit messages for
conventional commits, the intended update will be marked with the
commit itself for yarn to use later.